cypress/fixtures/fireboltMocks/acknowledgeChallenge.json

```diff
@@ -0,0 +1,34 @@
+{
+    "ACKNOWLEDGE_CHALLENGE_DENY": {
+        "method": "ackchallenge",
+        "response": [{
+            "isCancelled": true,
+            "withUi": true,
+            "error":{}
+        }]
+    },
+    "ACKNOWLEDGE_CHALLENGE_GRANTED": {
+        "method": "ackchallenge",
+        "response": [{
+            "isCancelled": false,
+            "withUi": true,
+            "result":{}
+        }]
+    },
+    "ACKNOWLEDGE_DEFERRED": {
+        "method": "ackchallenge",
+        "response": [{
+            "isCancelled": true,
+            "withUi": true,
+            "isExit": true,
+            "result":{}
+        }]
+    },
+    "ACKNOWLEDGE_TIMEOUT": {
+        "method": "ackchallenge",
+        "response": [{
+            "withUi": true,
+            "result":{}
+        }]
+    }
```

Comment:
isn't the fireboltMocks applicable to Ripple only? what is the guarantee that all platforms would use this format for mocking data ?

Reply:
Is there a readme document ? check how the ticket was implemented for fireboltMock.